src/apps/reservation-list/modal/delete-reservation/delete-reservation-modal.tsx

```diff
@@ -7,65 +7,64 @@ import {
   useDeleteReservations,
   getGetReservationsV2QueryKey
 } from "../../../../core/fbs/fbs";
-import { useDeleteV1UserReservationsIdentifier } from "../../../../core/publizon/publizon";
-import { ReservationType } from "../../../../core/utils/types/reservation-type";
-import { isDigital } from "../../../loan-list/utils/helpers";
+import {
+  getGetV1UserReservationsQueryKey,
+  useDeleteV1UserReservationsIdentifier
+} from "../../../../core/publizon/publizon";
+import { isFaust, isIdentifier } from "../../../dashboard/util/helpers";
 
 interface DeleteReservationModalProps {
   modalId: string;
-  reservation: ReservationType;
+  reservations: string[];
```

Comment:
As mentioned above: Please look into stronger typing here.

If we can not pass an array of `ReservationType` then look into an array of reservation ids. In that case naming should also be clarified.